WireAuthentication/Sources/WireAuthenticationLogic/SSO/SSOLoginVerificationToken.swift

```diff
@@ -39,34 +45,18 @@ package struct SSOLoginVerificationToken: Codable, Equatable {
         self.timeToLive = timeToLive
     }
 
-}
-
-package extension SSOLoginVerificationToken {
+    /// Whether the token is no langer valid (older than its time to live).
 
-    private static let defaultsKey = "CompanyLoginVerificationTokenDefaultsKey"
-
-    /// Stores the token in the provided defaults.
-    /// - parameter defaults: The defaults to store the token in.
-    /// - returns: Whether the write operation succeeded.
-    @discardableResult
-    func store(in defaults: UserDefaults) -> Bool {
-        do {
-            let data = try JSONEncoder().encode(self)
-            defaults.set(data, forKey: SSOLoginVerificationToken.defaultsKey)
-            return true
-        } catch {
-            return false
-        }
+    var isExpired: Bool {
+        abs(creationDate.timeIntervalSinceNow) >= timeToLive
```

Comment:
optional: I usually try to not have any logic in a simple data container.
I would have a separate validator which uses our `CurrentDateProviding` protocol in order to be easy mockable.
Feel free to leave it for now.

Reply:
ok I will create a ticket